code/inc/gfx/element/gfx_camera.hpp

```diff
@@ -0,0 +1,259 @@
+            /**
+            * \brief 
+            * 
+            * \param[in] 
+            * 
+            * \retval 
+            */
+            glm::vec3 GetRight(void) const;
+
+            /**************************************************************************************************/
+            /**
+            * \brief 
+            * 
+            * \param[in] 
+            * 
+            * \retval 
+            */
+            glm::vec3 GetForward(void) const;
+
+            /**************************************************************************************************/
+            /**
+            * \brief 
+            * 
+            * \param[in] 
+            * 
+            * \retval 
+            */
+            glm::quat GetDirection(void) const;
+
+            /**************************************************************************************************/
+            /**
+            * \brief 
+            * 
+            * \param[in] 
+            * 
+            * \retval 
+            */
+            glm::mat4 GetViewMatrix(void) const;
+
+            /**************************************************************************************************/
+            /**
+            * \brief 
+            * 
+            * \param[in] 
+            * 
+            * \retval 
+            */
+            void OnMouseWheel(float64_t delta);
+
+            /**************************************************************************************************/
+            /**
+            * \brief 
+            * 
+            * \param[in] 
+            * 
+            * \retval 
+            */
+            void Reset(void);
+
+            /**************************************************************************************************/
+            /**
+            * \brief 
+            * 
+            * \param[in] 
+            * 
+            * \retval 
+            */
+            void OnMouseMove(float64_t x, float64_t y, Input::EInputButton button);
+
+            /**************************************************************************************************/
+            /**
+            * \brief 
+            * 
+            * \param[in] 
+            * 
+            * \retval 
+            */
+            void UpdateViewMatrix(void);
+
+        private:
+            /*********************************/
+            // Private member variables
+            /*********************************/
+
+            glm::mat4 m_ViewMatrix;
+            glm::mat4 m_Projection;
+            glm::vec3 m_Position;
+
+            glm::vec3 m_Focus;
+
+            float32_t m_Distance;
+            float32_t m_Aspect;
+            float32_t m_FOV;
+            float32_t m_Near;
+            float32_t m_Far;
+
+            float32_t m_Pitch;
+            float32_t m_Yaw;
+
+            glm::vec2 m_CurrentPos2D;
```

Comment:
Add comments to these member variables

Reply:
Fixed https://github.com/tdiego1/worldweaver/pull/19/commits/1a0df967e70f1752177ebca99e4618e4ec06c26d